pipeline/outputs/s3.md

```diff
@@ -25,6 +25,9 @@ Records are stored in files in S3 as newline delimited JSON.
 | role\_arn | ARN of an IAM role to assume \(ex. for cross account access\). | None |
 | endpoint | Custom endpoint for the S3 API. | None |
 | sts\_endpoint | Custom endpoint for the STS API. | None |
+| canned\_acl | [Predefined Canned ACL policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl) for S3 objects. | None |
+| compression | Compression type for S3 objects. 'gzip' is currently the only supported value. The Content-Encoding HTTP Header will be set to 'gzip'. Compression is enabled when `use_put_object` is on. | None |
+| content\_type | A standard MIME type for the S3 object; this will be set as the Content-Type HTTP header. This option is enabled when `use_put_object` is on. | None |
```

Comment:
I would change to: "This option can be enabled when"...

If you say "is enabled" then that implies that compression is turned on when use_put_object is on, which I think is not true